packages/components/src/internal/app/utils.ts

```diff
@@ -30,36 +30,42 @@ import {
 export function initWebSocketListeners(store, notificationListeners?: string[], menuReloadListeners?: string[]): void {
     // register websocket listener for the case where a user logs out in another tab
     LABKEY.WebSocket.addServerEventListener('org.labkey.api.security.AuthNotify#LoggedOut', function (evt) {
```

Comment:
Instead of listen on 'org.labkey.api.security.AuthNotify#LoggedOut', it seems to me we should really listen on 1000 or 1003 . and then also check evt.reason === "org.labkey.api.security.AuthNotify#LoggedOut" , if we want to be consistent with dom/WebSocket.js . I don't know how reliable it is to check wasClean, but it's associated with websocket close events only, it seems. 

AuthNotify.LoggedOut is fired for user's all sessions (which doesn't seems to be used, not sure the reason why it was there @labkey-kevink. Could we change it to a different Enum, such as AuthNotify.LoggedOutOtherSession?). But onclose is only fired for the current sessions. And that's how dom/WebSocket.js handles labkey server log out events, but listerning to close event code: 1000 or 1003. 
 
            ```
// notify websocket clients associated with this http session, the user has logged out
            NotificationService.get().closeServerEvents(user.getUserId(), session, AuthNotify.LoggedOut);

            // notify any remaining websocket clients for this user that were not closed that the user has logged out elsewhere
            NotificationService.get().sendServerEvent(user.getUserId(), AuthNotify.LoggedOut);
```

Reply:
Using LoggedOutOtherSession to notify the other sessions seems like a good idea.  I'm not sure why we are notifying other sessions about the logout event, other than to match the login notification that is sent to the other sessions.

Reply:
@XingY @labkey-kevink Thank you both for the feedback on this PR. I think a new event for the session logout vs other session logout makes sense. For consistency, I decided to leave the all sessions event for log in and out the same (i.e. LoggedIn and LoggedOut) and to add a new event for the sessions logout event (i.e. SessionLogOut). Let me know if you think these event names should be updated.